rmgpy/data/kinetics/family.py

```diff
@@ -3868,3 +4221,53 @@ def getObjectiveFunction(kinetics1,kinetics2,obj=informationGain,T=1000.0):
     
     return obj(ks1,ks2), N1 == 0
 
+def makeRule(rr):
+    """
+    function for parallelization of rule and uncertainty calculation
+    Errors in Ln(k) at each reaction are treated as samples from a weighted normal distribution
+    weights are inverse variance weights based on estimates of the error in Ln(k) for each individual reaction
+    """
+    recipe,rxns,Tref,fmax,label,ranks = rr
+    N = len(rxns)
+    for i,rxn in enumerate(rxns):
+        rxn.rank = ranks[i]
+    rxns = np.array(rxns)
+    if N > 0:
+        kin = ArrheniusBM().fitToReactions(rxns,recipe=recipe)
+        if N == 1:
+            kin.uncertainty = RateUncertainty(mu=0.0,var=(np.log(fmax)/2.0)**2,N=1,Tref=Tref,correlation=label)
+        else:
+            dlnks = np.array([np.log(ArrheniusBM().fitToReactions(rxns[list(set(xrange(len(rxns)))-set([i,]))],recipe=recipe).toArrhenius(rxn.getEnthalpyOfReaction(Tref)).getRateCoefficient(T=Tref)/rxn.getRateCoefficient(T=Tref)) for i,rxn in enumerate(rxns)])
+            vars = (np.array([rank_accuracy_map[rxn.rank].value_si for rxn in rxns])/(2.0*8.314*Tref))**2
```

Comment:
`vars` shadows a built-in function. I would suggest renaming it.

Reply:
fixed